src/app/(auth)/sign-up/page.tsx

```
@@ -1,6 +1,18 @@
+import { redirect } from 'next/navigation'
+
+import { auth } from '@/lib/auth'
 import { SignUpView } from '@/modules/auth/ui/views/sign-up-view'
 
-const Page = () => {
+const Page = async () => {
+	const session = await auth.api.getSession({
+		headers: await headers(),
+	})
+
+	if (!!session) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)